packages/resource-deployment/scripts/wait-for-deployment.sh

```diff
@@ -28,7 +28,7 @@ for arg in "$@"; do
     previousFlag=$arg
 done
 
-if [ -z $resourceName ] || [ -z $timeout ]; then
+if [ -z "$resourceName" ] || [ -z "$timeout" ]; then
```

Comment:
curious why the `""` are required only here? should this look like `if [[ ... ]]`